controllers/mandataris.ts

```diff
@@ -203,3 +208,24 @@ export async function handleBulkSetPublicationStatus(
     STATUS_CODE.BAD_REQUEST,
   );
 }
+
+async function generate(config): Promise<Array<string>> {
```

Comment:
maybe call this `generateRows` to mirror the thing that's going on in the frontend. Else generate feels a bit unclear to me

Reply:
Updated it to _generateRows_